### ISSUE TITLE
Generation of module

### ISSUE THREAD

**Issue by ersagun** [compodoc/compodoc#442]
##### **Overview of the issue**
Error :  <rejected> { SyntaxError: Unexpected '(' at line 1 column 29 of the JSON5 data. Still to read: "()=>\"ClientModule\"},"

##### **Operating System, Node.js, npm, compodoc version(s)**
Windows 8

##### **Angular configuration, a `package.json` file in the root folder**
Angular 5

##### **Compodoc installed globally or locally ?**
globally and locally

**Comment by vogloblinsky** [comment]
It is possible to have the code of your ClientModule file ?

**Comment by ersagun** [comment]
Hi thank you for your help, I solved the problem :

I think the problem was cming from this line : path: '', loadChildren: () => ClientComponent

When I declared it like : path: '', loadChildren: "./client/client.module#ClientModule"

I parsed the file normally.

Thank you for your helps. I recommand people who encounter this problem to execute "npm run build" systematically before trying compodoc command. Npm start launch the app but doesn't find some syntactical problems.
